.pipelines/upgrade-e2e-job.yaml

```diff
@@ -10,9 +10,9 @@ pr:
   branches:
     include:
     - main
-  paths:
-    include:
-      - charts/osm-arc/Chart.yaml
+  #paths:
```

Comment:
can we delete these

Reply:
I'm going to recomment this before I merge, I just commented this out temporarily so that I would be able to trigger the upgrade job on this PR.